GOVERNANCE.md

```diff
@@ -0,0 +1,97 @@
+# Governance
+
+`haskell-cryptography` (herein referred to as the "Organisation") is a GitHub organisation that aims to provide a support structure to open-source maintainers producing cryptography projects.
+
+The Organisation has the following goals:
+
+* Prevent situations where too few people are supporting critical projects for the ecosystem.
+* Prevent having a closed group of people having too much control over every aspect of a project.
+* To encourage and support part-time, niche contributors to the projects.
+* To ensure continuity and stability through documentation, testing and benchmarking.
+* To listen to the needs of the community and industrial users, and lowering the barrier to industrial adoption.
+* To ensure that current security practices are easy for others and maintained whithin the ecosystem.
+* To ensure timely update in the community package repository
```

Comment:
By which I assume you mean Hackage? If so, better say it, or at least list as an example. Also, 'timely updates'.